.azure/windows-stack.yml

```diff
@@ -0,0 +1,66 @@
+jobs:
+- job: Windows_Stack
+  timeoutInMinutes: 0
+  pool:
+    vmImage: windows-2019
+  strategy:
+    matrix:
+      stack-def:
+        YAML_FILE: stack.yaml
+      stack-8.6.5:
+        YAML_FILE: stack-8.6.5.yaml
+      stack-8.6.4:
+        YAML_FILE: stack-8.6.4.yaml
+      stack-8.6.2:
+        YAML_FILE: stack-8.6.2.yaml
+      stack-8.6.1:
+        YAML_FILE: stack-8.6.1.yaml
+      stack-8.4.4:
+        YAML_FILE: stack-8.4.4.yaml
+      stack-8.4.3:
+        YAML_FILE: stack-8.4.3.yaml
+      stack-8.4.2:
+        YAML_FILE: stack-8.4.2.yaml
+      stack-8.2.2:
+        YAML_FILE: stack-8.2.2.yaml
+  steps:
+  - bash: |
+      git submodule sync
+      git submodule update --init
+    displayName: Sync submodules
+  - bash: |
+      export STACK_ROOT="$(Build.SourcesDirectory)"/.stack-root
+      curl -sSkL http://www.stackage.org/stack/windows-i386 -o /usr/bin/stack.zip
```

Comment:
Hi, why use windows-i386 instead x86_64?, i think the last one is the most used lately

Reply:
Even there is no ghc-8.6.4 for win32, it is strange

Reply:
It is strange indeed! Maybe we can raise an issue? or find an associated issue?

Reply:
Anyway i would use 64 bits arch by default, like in linux builds, it is the most common arch nowadays